src/test/java/org/gridsuite/study/server/service/dynamicsimulation/DynamicSimulationServiceTest.java

```diff
@@ -82,35 +84,25 @@ public class DynamicSimulationServiceTest {
 
     private static final String VARIANT_1_ID = "variant_1";
 
-    private static final String STUDY_UUID_STRING = "00000000-0000-0000-0000-000000000000";
-    private static final UUID STUDY_UUID = UUID.fromString(STUDY_UUID_STRING);
+    private static final UUID STUDY_UUID = UUID.randomUUID();
 
     // converged node
-    private static final String NETWORK_UUID_STRING = "11111111-0000-0000-0000-000000000000";
-    public static final UUID NETWORK_UUID = UUID.fromString(NETWORK_UUID_STRING);
-
-    private static final String NODE_UUID_STRING = "22222222-0000-0000-0000-000000000000";
-    public static final UUID NODE_UUID = UUID.fromString(NODE_UUID_STRING);
-
-    public static final String RESULT_UUID_STRING = "99999999-0000-0000-0000-000000000000";
-    public static final UUID RESULT_UUID = UUID.fromString(RESULT_UUID_STRING);
-
-    public static final String TIME_SERIES_UUID_STRING = "77777777-0000-0000-0000-000000000000";
-    public static final UUID TIME_SERIES_UUID = UUID.fromString(TIME_SERIES_UUID_STRING);
-
-    public static final String TIME_LINE_UUID_STRING = "88888888-0000-0000-0000-000000000000";
-    public static final UUID TIME_LINE_UUID = UUID.fromString(TIME_LINE_UUID_STRING);
+    public static final UUID NETWORK_UUID = UUID.randomUUID();
+    public static final UUID NODE_UUID = UUID.randomUUID();
+    public static final UUID RESULT_UUID = UUID.randomUUID();
+    public static final UUID TIME_SERIES_UUID = UUID.randomUUID();
+    public static final UUID TIME_LINE_UUID = UUID.randomUUID();
 
     // running node
-    private static final String NODE_UUID_RUNNING_STRING = "22222222-1111-0000-0000-000000000000";
-    public static final UUID NODE_UUID_RUNNING = UUID.fromString(NODE_UUID_RUNNING_STRING);
-
-    public static final String RESULT_UUID_RUNNING_STRING = "99999999-1111-0000-0000-000000000000";
-    public static final UUID RESULT_UUID_RUNNING = UUID.fromString(RESULT_UUID_RUNNING_STRING);
+    public static final UUID NODE_UUID_RUNNING = UUID.randomUUID();
+    public static final UUID RESULT_UUID_RUNNING = UUID.randomUUID();
 
     public static final String TIME_SERIES_NAME_1 = "NETWORK__BUS____2-BUS____5-1_AC_iSide2";
     public static final String TIME_SERIES_NAME_2 = "NETWORK__BUS____1_TN_Upu_value";
-    public static final String TIME_LINE_NAME = "TimeLine";
+    public static final String TIMELINE_NAME = "Timeline";
+
+    public static final UUID REPORT_UUID = UUID.randomUUID();
+    public static final String REPORT_ID = NODE_UUID.toString();
```

Comment:
If you wan to match the dto ReportInfos :
```suggestion
    public static final String REPORTER_ID = NODE_UUID.toString();
```

Reply:
DONE